package.json

```diff
@@ -48,7 +48,7 @@
     "lodash": "^4.17.11"
   },
   "peerDependencies": {
-    "vue": "^2.2.2"
+    "vue": "^2.4.0"
```

Comment:
Do you have an idea of how many users it would impact?

Reply:
likely none, it's a quite old version and this is a major change anyway :)

Reply:
(also we already required it via another thing earlier 😉, it just wasn't changed yet)